src/enums.js

```diff
@@ -28,8 +28,23 @@ export const MIN_ANGLE_POINTS_DISTANCE = 0.7;
 export const MEASUREMENT_LABELS_GROUP_NAME = "Measure_Labels";
 export const ANGLE = "ANGLE";
 export const LABELS_GROUP_NAME = "Labels_Group";
+export const ELEMENT_LABELS_GROUP_NAME = "Element_Labels_Group";
+export const COORDINATE_LABELS_GROUP_NAME = "Coordinate_Labels_Group";
 
 export const COLORS = {
     RED: 0xff0000,
     GREEN: 0x00ff00,
 };
+
+export const MEASUREMENT_MODES = {
+    NONE: "none",
```

Comment:
It should not be needed. We should have something like - enabled/disabled

Reply:
We could also define such variables as TS enums